jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/MillisecondsToSecondsConverter.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.pr.client.editors.instance.diagram;
+
+import org.jboss.errai.databinding.client.api.Converter;
+
+public class MillisecondsToSecondsConverter implements Converter<Long, String> {
```

Comment:
Shouldn't be the converter methods toModelValue and toWidgetValue inverse functions?

Reply:
the model brings value in milis but the update method takes seconds, so its just one way. Once its been converted to seconds it remains like that.